test/component/test_component.py

```diff
@@ -105,4 +104,4 @@ def init_rabbitmq(self, rabbitmq_amqp=RABBIT_AMQP,
         channel.queue_bind(exchange=exchange_name, queue=queue_name, routing_key=binding_key)
         self.channel = channel
         self.queue_declare_result = queue_declare_result
-        return
+        return
```

Comment:
Should have a newline at the end of every py file (otherwise flake8 will complain).

Reply:
fixed (travis already told about this one :) )